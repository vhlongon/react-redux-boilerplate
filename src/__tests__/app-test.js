import React from 'react';
import App from '../js/components/app';
import { shallow } from 'enzyme';
import {mockActiveItem} from './mock';

function setup() {
  const props = {
    title: 'React redux boilerplate test',
  };

  const enzymeWrapper = shallow(<App {...props} />)

  return {
    props,
    enzymeWrapper
  }
}

describe('components', () => {
  describe('App', () => {
    test('should render self and subcomponents', () => {
       const { enzymeWrapper } = setup();
       expect(enzymeWrapper.text()).toMatchSnapshot();
    });
  });
});
