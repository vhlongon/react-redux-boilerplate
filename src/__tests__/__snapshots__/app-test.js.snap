exports[`components App should render self and subcomponents 1`] = `"React redux boilerplate test<Connect(ItemsList) /><Connect(Component) />"`;
